shared/src/business/useCases/pendingItems/generatePrintablePendingReportInteractor.js

```diff
@@ -24,13 +24,14 @@ exports.generatePrintablePendingReportInteractor = async ({
     throw new UnauthorizedError('Unauthorized');
   }
 
+  //TODO: prefereably decodeURIComponent in the router/utility method for easy mocking
```

Comment:
is there a related card in the backlog for this work?

Reply:
Yes! https://trello.com/c/BcRg2BzP